pkg/plugins/autodiscovery/dockerfile/dockerfile.go

```diff
@@ -19,8 +19,178 @@ var (
 		"Dockerfile",
 		"Dockerfile.*",
 	}
+	// GlobalIgnore specifies a list of globally ignored docker image, like the `scratch` image which
+	// never needs to be updated as it's not a "real" image
+	ScratchIgnore MatchingRule = MatchingRule{
```

Comment:
Is this variable used outside odf the dockerfile package?
Maybe it can be private?